client_test.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/go-redis/redis"
```

Comment:
We need to add that to `dep`. Is there a flag to also mark it a development dependency?